src/jit/compiler.cpp

```diff
@@ -2403,31 +2403,32 @@ unsigned ReinterpretHexAsDecimal(unsigned in)
     return result;
 }
 
-void Compiler::compInitOptions(CORJIT_FLAGS* jitFlags)
+void Compiler::compInitOptions(JitFlags* jitFlags)
 {
 #ifdef UNIX_AMD64_ABI
     opts.compNeedToAlignFrame = false;
 #endif // UNIX_AMD64_ABI
     memset(&opts, 0, sizeof(opts));
 
-    unsigned compileFlags = jitFlags->corJitFlags;
-
     if (compIsForInlining())
     {
-        assert((compileFlags & CORJIT_FLG_LOST_WHEN_INLINING) == 0);
-        assert(compileFlags & CORJIT_FLG_SKIP_VERIFICATION);
+        assert(!jitFlags->IsSet(JitFlags::JIT_FLAG_BBOPT));
```

Comment:
That's possible, but I was hoping that JitFlags would be a thin adapter class, and this would add more functionality to it than exists in the CORJIT_FLAGS class. Another option would be to create a single "global" or static `CORJIT_FLAGS` object, say `JitFlagsLostWhenInlining` and initialize it to the set, then use the appropriate CORJIT_FLAGS Add/Remove functions on the entire set.